app/src/main/java/io/github/muntashirakon/AppManager/appops/AppOpsManager.java

```diff
@@ -108,6 +108,7 @@
             "deny",         // MODE_ERRORED
             "default",      // MODE_DEFAULT
             "foreground",   // MODE_FOREGROUND
+            "ask",          // MODE_ASK (MIUI)
```

Comment:
Define `MODE_ASK` as a constant below `MODE_FOREGROUND` in line 103 and add it in the `IntDef` list below line 50.